pkg/auth/opaauther_test.go

```diff
@@ -72,150 +71,8 @@ func Test_opa_authorize_with_permissions(t *testing.T) {
 					Methods: []string{"/metalstack.api.v2.UnknownService/Get"},
 				},
 			},
-			wantErr: connect.NewError(connect.CodePermissionDenied, fmt.Errorf("method denied or unknown: /metalstack.api.v2.UnknownService/Get")),
+			wantErr: errorutil.PermissionDenied("method denied or unknown: /metalstack.api.v2.UnknownService/Get"),
 		},
-		// {
-		// 	name:    "cluster get not allowed, no token",
-		// 	subject: "john.doe@github",
-		// 	method:  "/metalstack.api.v2.IPService/Get",
-		// 	req:     v1.ClusterServiceGetRequest{},
-		// 	userJwtMutateFn: func(t *testing.T, jwt string) string {
-		// 		return ""
```

Comment:
Maybe these tests were ensuring some basic authorization behavior and it would be worth to carry them over using another entity like machine instead of cluster?